src/coreclr/src/debug/createdump/crashinfomac.cpp

```diff
@@ -30,45 +30,6 @@ CrashInfo::CleanupAndResumeProcess()
     }
 }
 
-static
```

Comment:
I think this is because he did a `git revert` and then made the same changes in another branch or reused the commits when he merged.  Any idea how to undo this removal?  Is it just a matter of removing his commits from this PR instead of just `revert`ing them?

Reply:
I'd cherry pick them out, yet.

Reply:
Sorry about that.

Reply:
I meant to remove all of those changes.

Reply:
`git rebase -i` helped me remove the commits entirely 👍